lib/types/measures/Measure.ts

```diff
@@ -0,0 +1,80 @@
+import { JSONObject } from 'kuzzle';
+import { KDocumentContent } from 'kuzzle-sdk';
+
+import { MeasureUnit } from './MeasureDefinition';
+import { MeasureOrigin } from './MeasureOrigin';
+
+/**
+ * Represents a measurement of a value to post on an asset
+ */
+export interface BaseAssetMeasure extends KDocumentContent {
```

Comment:
As we discussed:
 - a `MeasureContent` represents the content of the entire measure document as it is stored in the database
 - a `Measurement` represents the actual measure 

And finally, the measure name will always be present, not only for the asset (see https://github.com/kuzzleio/kuzzle-device-manager/issues/168#issuecomment-1142182855)